build_tools/github_actions/test_executable_scripts/test_hipcub.py

```diff
@@ -0,0 +1,24 @@
+import os
+import subprocess
+from pathlib import Path
+
+THEROCK_BIN_DIR = os.getenv("THEROCK_BIN_DIR")
+SCRIPT_DIR = Path(__file__).resolve().parent
+THEROCK_DIR = SCRIPT_DIR.parent.parent.parent
+
+subprocess.run(
+    [
+        "ctest",
+        "--test-dir",
+        f"{THEROCK_BIN_DIR}/hipcub",
+        "--output-on-failure",
+        "--parallel",
+        "8",
+        "--timeout",
+        "300",
+        "--repeat",
+        "until-pass:3",
+    ],
+    cwd=THEROCK_DIR,
+    check=True,
+)
```

Comment:
Oh wait. Do these scripts log the commands they are running? They should. Can use the `exec` helper function from other parts of the codebase.

Reply:
Can you link to some logs? Might be okay depending on how "hidden".

Reply:
That expands to `python build_tools/github_actions/test_executable_scripts/test_rocblas.py`, when I would want to see this translated to a command I could run myself outside of a script:
```python
subprocess.run(
    [
        f"{THEROCK_BIN_DIR}/rocblas-test",
        "--yaml",
        f"{THEROCK_BIN_DIR}/rocblas_smoke.yaml",
    ],
    cwd=THEROCK_DIR,
    check=True,
)
```

like `.../path/to/rocblas-test --yaml .../path/to/rocblas_smoke.yaml`